Extensions/Today/TodayViewController.swift

```diff
@@ -60,35 +61,42 @@ class TodayViewController: UIViewController, NCWidgetProviding {
         imageButton.addTarget(self, action: #selector(onPressNewPrivateTab), forControlEvents: .touchUpInside)
         imageButton.label.text = TodayStrings.NewPrivateTabButtonLabel
         let button = imageButton.button
-        button.frame = CGRect(width: 60.0, height: 60.0)
-        button.performGradient(colorOne: TodayUX.privateSearchButtonColorFaintDarkPurple, colorTwo: TodayUX.privateSearchButtonColorDarkPurple, colorThree: TodayUX.privateSearchButtonColorBrightPurple)
-        button.layer.cornerRadius = button.frame.size.width/2
-        button.clipsToBounds = true
-        button.setImage(UIImage(named: "quick_action_new_private_tab")?.withRenderingMode(.alwaysTemplate), for: .normal)
-        button.tintColor = UIColor.white
-
+        button.setImage(UIImage(named: "private-search")?.withRenderingMode(.alwaysOriginal), for: .normal)
+        button.imageView?.contentMode = .scaleAspectFit
         let label = imageButton.label
-        label.tintColor = UIColor(named: "widgetLabelColors")
-        label.textColor = UIColor(named: "widgetLabelColors")
+        if #available(iOS 13, *) {
+            label.textColor = UIColor(named: "widgetLabelColors")
+            label.tintColor = UIColor(named: "widgetLabelColors")
+        } else {
+            label.textColor = TodayUX.labelColor
+            label.tintColor = TodayUX.labelColor
+        }
         label.font = UIFont.systemFont(ofSize: TodayUX.imageButtonTextSize)
         imageButton.sizeToFit()
         return imageButton
     }()
 
     fileprivate lazy var openCopiedLinkButton: ButtonWithSublabel = {
         let button = ButtonWithSublabel()
-
         button.setTitle(TodayStrings.GoToCopiedLinkLabel, for: .normal)
         button.addTarget(self, action: #selector(onPressOpenClibpoard), for: .touchUpInside)
-
         // We need to set the background image/color for .Normal, so the whole button is tappable.
         button.setBackgroundColor(UIColor.clear, forState: .normal)
         button.setBackgroundColor(TodayUX.backgroundHightlightColor, forState: .highlighted)
-
-        button.setImage(UIImage(named: "copy_link_icon")?.withRenderingMode(.alwaysTemplate), for: .normal)
-
+        button.setImage(UIImage(named: "copy_link_icon")?.withRenderingMode(.alwaysOriginal), for: .normal)
         button.label.font = UIFont.systemFont(ofSize: TodayUX.labelTextSize)
         button.subtitleLabel.font = UIFont.systemFont(ofSize: TodayUX.linkTextSize)
+        if #available(iOS 13, *) {
+            button.label.textColor = UIColor(named: "widgetLabelColors")
```

Comment:
color set cleanup: 
```
        button.label.textColor = TodayUX.labelColor
        button.label.tintColor = TodayUX.labelColor
        button.subtitleLabel.textColor = TodayUX.subtitleLabelColor
        button.subtitleLabel.tintColor = TodayUX.subtitleLabelColor

Reply:
defined the label color variables differently for now to avoid the clutter of supporting earlier versions of iOS.